### ISSUE TITLE
stack, service resources

### ISSUE THREAD

**Issue by matti** [kontena/terraform-provider-kontena#21]
kubernetes provider has replicaset/pod resources.

these would be super handy to describe initial grid state like kontena/lb etc

**Comment by SpComb** [comment]
Services would probably be the next step, but stacks will be a bit limited... the native Go `kontena` provider won't be able to support stack YAMLs, that would need some separate provider that runs the ruby CLI.

**Comment by matti** [comment]
yep, I'm gonna do that local-exec module at some point as a workaround
